fe/fe-core/src/main/java/org/apache/doris/analysis/ShowTableStatsStmt.java

```diff
@@ -75,8 +75,10 @@ public class ShowTableStatsStmt extends ShowStmt implements NotFallbackInParser
             new ImmutableList.Builder<String>()
             .add("table_name")
             .add("index_name")
-            .add("row_count")
+            .add("analyze_row_count")
             .add("update_time")
```

Comment:
update_time:
what the meaning of this time, the update time of analyze_row_count or report_row_count?

Reply:
The time we collect the row count. This is used for the 4 minutes check to decide use analyze row count or report row count.

Reply:
so, it is better to rename to "last_analyze_time"

Reply:
removed